sdk/deviceupdate/Azure.IoT.DeviceUpdate/src/Azure.IoT.DeviceUpdate.csproj

```diff
@@ -1,22 +1,24 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AssemblyTitle>Device Update for IoT Hub client library</AssemblyTitle>
-    <Version>1.0.0-beta.3</Version>
+    <Version>1.0.0-beta.1</Version>
```

Comment:
A package with this name has already been released to NuGet: https://www.nuget.org/packages/Azure.Iot.DeviceUpdate/1.0.0-beta.1.  We will need to use 1.0.0-beta.3.